### ISSUE TITLE
aws-lambda-java-events:2.0 missing 'path' in  ProxyRequestContext

### ISSUE THREAD

**Issue by DALDEI** [aws/aws-lambda-java-libs#28]
The class ProxyRequestContext, in lambda API Events is missing the 'path' member.
Attempting to data map a request gives 
`
2017-10-11 07:23:32 ERROR SMSReply:81 - com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "path" (class com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent$ProxyRequestContext), not marked as ignorable (8 known properties: "resourcePath", "apiId", "accountId", "resourceId", "requestId", "stage", "httpMethod", "identity"])`

Except from payload:
```

 "requestContext": {
        "path": "/Prod",
        "accountId": "114709661826",
        "resourceId": "epap6azisg",
        "stage": "Prod",
        "requestId": "12e30354-ae55-11e7-9a45-9d5723258a9f",
        "identity": {
            "cognitoIdentityPoolId": null,
            "accountId": null,
            "cognitoIdentityId": null,
            "caller": null,
            "apiKey": "",
            "sourceIp": "xx.xx.xx.xx",
            "accessKey": null,
            "cognitoAuthenticationType": null,
            "cognitoAuthenticationProvider": null,
            "userArn": null,
            "userAgent": "XX/1.1",
            "user": null
        },


```

**Comment by asuresh8** [comment]
Thanks for pointing this out. We'll work on adding this field.

**Comment by carlzogh** [comment]
The `path` field was added in https://github.com/aws/aws-lambda-java-libs/commit/f0114f6862c540fe4eea0587395568e9a78a9c7b which was released as part of version `2.0.2`.
Thanks for reporting this!
